docs/styling.md

```diff
@@ -16,6 +16,7 @@ The available variables are:
 Explorer location   | Variables
 --------------------|--------------------
 Fonts               | `--font-regular` `--font-mono`, `--font-size-small`, `--font-size-regular`, `--font-size-mono`
+Schema Table View   | `--schema-table-key-width`, `--schema-table-type-width`
```

Comment:
Can we take out the `-table` part? Reason being that keeping the number of variables at a minimum is a huge plus, which let's us reuse this effectively in schema tree, if necessary.

But I want to have a slightly longer discussion about if there is an alternative to getting this right. I can imagine it won't be easy/obvious for someone to change these values so that they are happy no matter what. I'm wondering if there is something else we can do.

For my understanding, can you share an example schema that contains this issue so we can figure out what alternatives could make sense here? I would love it if this could be automatically handled, assuming we wrote some code like that, any ideas on what that would be? An example would do wonders for pondering that answer.